app/models/podcast_import.rb

```diff
@@ -339,6 +343,7 @@ def create_episode(story, entry)
     create_attributes[:keywords] = (entry[:itunes_keywords] || '').split(',').map(&:strip)
     create_attributes[:position] = entry[:itunes_order]
     create_attributes[:url] = episode_url(entry) || distro.default_url(story)
+    create_attributes[:itunes_type] = entry[:itunes_episode_type] unless entry[:itunes_episode_type].blank?
```

Comment:
Why the blank check here, but not in the other fields?

Reply:
no particular reason, really -- just that the other fields were inside a call to `create`